recipes/Beef/EasyBeefStew.recipe

```diff
@@ -0,0 +1,28 @@
+name: Easy Beef Stew
+prepTime: 20 minutes
+cookTime: 3 hours
+portions: 6
+hint: You can add any aromatics you wish. Thyme, oregano, and parsley are my personal favorites.
+
+ingredients:
+  - (3 lb)    Chuck beef, cut into 1 inch thick steaks
+  - (1)       Large white onion, sliced
+  - (1 lb)    Potatoes, cut into 1 inch cubes
+  - (2 cups)  Baby carrots
+  - (16 oz)   Beer (stout)
+  - (2 cups)  Chicken broth
+  - (1 Tbsp)  Worcestershire sauce
+  - (4 oz)    Frozen peas
+  - (1 tsp)   Salt
+  - (1 tsp)   Cracked pepper
+  - (1 Tbsp)  Flour
+steps:
+  - Preheat oven to 300F
+  - Sear the beef steaks in a hot pan with some oil until a crust develops. Set
+      aside on a plate to cool.
+  - Add onion, potatoes, carrots, beer, chichen broth, worcestershire sauce,
+      peas, salt, and pepper to a dutch oven.
+  - Bring to a simmer
+  - Cut beef into 1.5 inch cubes, toss with flour.
```

Comment:
Huh - out of curiosity, what's the effect of searing, then dredging, then braising? I've seen some that call for dredging before the searing (e.g. http://www.reluctantgourmet.com/flouring-meat/), but I don't really feel like I know what the big difference is. Perhaps the science of cooking or the food lab has the answer...?